src/components/Comment/Comment.tsx

```diff
@@ -116,15 +118,41 @@ const Row = ({ children }: { children: JSX.Element | JSX.Element[] }) => (
 export const Comment = ({
   comment,
   pillar,
-  setCommentBeingRepliedTo
+  setCommentBeingRepliedTo,
+  user
 }: Props) => {
   const commentControlsButtonStyles = commentControlsButton(pillar);
+  const [scopedComment, setScopedComment] = useState(comment);
```

Comment:
I don't think we should be loading the entire comment into state. This adds a lot of complexity to this component for a very limited benefit. It gives internal staff users immediate feedback on their actions but makes the codebase more complicated to reason about and increases the risk of bugs. Frontend doesn't have this feature - you only see updates after you refresh - so we have no requirement to actually add it